src/Components/Samples/BlazorServerApp/App.razor

```diff
@@ -1,4 +1,4 @@
-﻿<Router AppAssembly="@typeof(Program).Assembly" PreferExactMatches="true">
+﻿<Router AppAssembly="@typeof(Program).Assembly" PreferExactMatches="@true">
```

Comment:
This would indicate a bug in rzc or in the references that it gets to compute what the type of the parameter is. Can we figure out why that's happening?

Reply:
Dug into this further. This particular issue is actually rather old. It happened happened on builds that occurred on 12/11 on the template tests (I found this while looking through quarantined test stats). Is there a chance that it could be related to [this](https://github.com/dotnet/aspnetcore/pull/28285)?

It doesn't seem to be happening anymore. Although the test it originally occurred on isn't running anymore...

Reply:
> Is there a chance that it could be related to this?

Yup. We were probably compiling against the wrong reference assemblies.

> Although the test it originally occurred on isn't running anymore...

o.O?